### DISCUSSION TITLE
Why not av==11.*

### DISCUSSION THREAD

**Discussion by kusf** [SYSTRAN/faster-whisper#684] [General]
Is there any particular reason why we are using av==10.* instead of av==11.* in requirements.txt?

I was getting some issues installing av==10. 
I am not the only one getting this issue:
https://github.com/SYSTRAN/faster-whisper/issues/560
https://github.com/PyAV-Org/PyAV/issues/1140

Once I switched  av==10.* to of av==11.* in requirements.txt, everything worked fine

**Comment by trungkienbkhn** [comment]
Hello. Yes, we can upgrade av to version 11*. You can see this PR: https://github.com/SYSTRAN/faster-whisper/pull/679
